app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,42 @@
 <!DOCTYPE html>
 <html>
-  <head>
+ <head>
     <title>FrontEndRails</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
-  </head>
-
-  <body>
-    <%= yield %>
-  </body>
+</head>
+<body>
+  <% if !current_user.nil? %>
+    <nav class="navbar navbar-expand-xl bg-dark navbar-dark sticky-top" id="navbar-<%= current_user.id %>">
+      <a class="navbar-brand" href="/">Home</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+        <span class="navbar-toggler-icon"></span>
```

Comment:
@LHJE @dcoleman21 Home = Dashboard, right? If so, should we rename this nav item accordingly?

Reply:
Good idea!